x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/title_with_explanation.tsx

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
We have a similar [thing](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/table/column_header.tsx) in the hosts table for displaying column headers with tooltip

It's generic and doesn't require the creation of a component per column ([see usage](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/hooks/use_hosts_table.tsx#L283)). Would it be possible to replicate the same thing here?